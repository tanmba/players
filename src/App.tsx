import './App.css';
import Players from './Layout/PlayersList/Players';

function App() {
  return (
    <div className="App">
      <Players />
    </div>
  );
}

export default App;
