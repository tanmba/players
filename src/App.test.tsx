import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';

describe("App component", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<App />);

    expect(asFragment()).toMatchSnapshot();
  });
});