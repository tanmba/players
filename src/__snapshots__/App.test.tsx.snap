// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="homeBcg MuiBox-root css-1k6ezy6"
    >
      <div
        class="MuiFormControl-root MuiTextField-root css-3k4p6m-MuiFormControl-root-MuiTextField-root"
      >
        <label
          class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
          data-shrink="false"
          for=":r0:"
          id=":r0:-label"
        >
          Rechercher un joueur
        </label>
        <div
          class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
        >
          <input
            aria-invalid="false"
            class="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
            id=":r0:"
            type="text"
            value=""
          />
          <fieldset
            aria-hidden="true"
            class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
          >
            <legend
              class="css-1ftyaf0"
            >
              <span>
                Rechercher un joueur
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
